db/migrate/20191002124400_add_hierarchies_to_campy_events.rb

```diff
@@ -0,0 +1,12 @@
+#  Copyright (c) 2019, Pfadibewegung Schweiz. This file is part of
+#  hitobito_pbs and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_pbs.
+
+class AddHierarchiesToCampyEvents < ActiveRecord::Migration
+  def change
+    add_column :events, :parent_id, :integer, null: true
```

Comment:
Gibts einen Grund wieso hier nicht `add_reference` oder `add_foreign_key` verwendet wird?

Reply:
Hmm. normalerweise würde ich das jetzt ändern, weil es sprechender wäre. Da wir das aber eh noch zu einem nested_set erweitern lasse ich es erstmal so. Ich würde es sonst zu `add_belongs_to :events, :parent, index: true` ändern. Foreignkeys in der DB verwenden wir aktuell in hitobito überhaupt nicht, daher würde ich hier auch keinen setzen.

Reply:
Interessanterweise hat es seit noch nicht allzu langer Zeit bei den doorkeeper migrations foreign keys drin. Aber vielleicht braucht doorkeeper die.